examples/purchaseTesterTypescript/android/build.gradle

```diff
@@ -19,3 +19,11 @@ buildscript {
         classpath("com.facebook.react:react-native-gradle-plugin")
     }
 }
+
+allprojects {
+  repositories {
+    google()
+    mavenCentral()
+    mavenLocal()
+  }
```

Comment:
added this because I kept adding it myself after publishing to maven local, and I figured it doesn't hurt

Reply:
Might want to make this a separate PR so we can easily revert it if it creates issues, and so there's more visibility into this while looking at logs.

Reply:
https://github.com/RevenueCat/react-native-purchases/pull/859